src/doc/integrating_telio.md

```diff
@@ -131,6 +131,102 @@ telio.destroy()!!;
 
 </multi-code>
 
+
+### Feature config
+
+App can deserialize JSON string with feature config before passing to telio.
+
+<multi-code-select></multi-code-select>
+
+<multi-code>
+
+```rust no_run
+use telio::{ffi::*, types::*};
+use telio_model::features::FeatureLana;
+
+let json_feature_config = "<feature config fetched from API>".to_owned();
+let mut feature_config = string_to_features(json_feature_config).unwrap();
+
+feature_config.lana = Some(FeatureLana {  event_path: "some/path.db".to_owned(), prod: true });
+if let Some(nurse) = &mut feature_config.nurse {
+    nurse.fingerprint = "me".to_owned();
+}
+
+#[derive(Debug)]
+struct EventHandler;
+impl TelioEventCb for EventHandler {
+    fn event(&self, payload: String) {}
+}
+
+let telio = Telio::new(feature_config, Box::new(EventHandler)).unwrap();
+telio.destroy().unwrap();
+```
+
+```go
+import github.com/nordsecurity/telio
+
+json_feature_config := "<feature config fetched from API>";
+feature_config, err := string_to_features(json_feature_config);
```

Comment:
This is also StringToFeatures, go uses PascalCase for public members, and cammelCase for private ones